src/components/ClowdApp.tsx

```diff
@@ -1,6 +1,6 @@
 import * as React from 'react';
 //import { K8sResourceCommon, useK8sWatchResource } from '@openshift-console/dynamic-plugin-sdk';
-import { K8sResourceCommon} from '@openshift-console/dynamic-plugin-sdk';
+import { K8sResourceCommon, useK8sWatchResources} from '@openshift-console/dynamic-plugin-sdk';
```

Comment:
This is already imported in the line below. Is there a need for it to be imported again?

Reply:
probably not needed, it was add by the yarn build probably